localisation/00_zuwc_united_localisation.csv

```diff
@@ -3305,3 +3305,46 @@ awaken_paragon_desc;Ask the Klaxxi to awaken [Root.GetBestName].;;Bittet die Kla
+EVTOPTA_WCULD_100_superweapon;Now where's that superweapon everyone's talking about?;;;;;;;;;;;;;x
+EVTOPTA_WCULD_100_outsiders;We shall see what these outsiders bring to Uldum.;;;;;;;;;;;;;x
+EVTOPTA_WCULD_100_exploring;Fortune and glory, here I come.;;;;;;;;;;;;;x
+uldum_isolation_dont_block_interactions_tooltip;The Shroud of Uldum does NOT block interactions;;;;;;;;;;;;;x
+RULE_ULDUM_ISOLATION;�RWarCraft�!: Shroud of Uldum;;;;;;;;;;;;;x
+RULE_ULDUM_ISOLATION_ON;On;;;;;;;;;;;;;x
+RULE_ULDUM_ISOLATION_ON_DESC;The Shroud of Uldum is enabled, making Uldum isolated until the beginning of 600s.;;;;;;;;;;;;;x
+RULE_ULDUM_ISOLATION_OFF;Off;;;;;;;;;;;;;x
+RULE_ULDUM_ISOLATION_OFF_DESC;The Shroud of Uldum is disabled, making Uldum open to the world.;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10100;Agents of Twilight;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10100;Ever since you have embarked on the perilous journey to master the Void, you have been hearing ominous whispers. Not too long after, you started receiving numerous dreams that made little to no sense to you, but nevertheless they seemed very real. Today seems no different, but the contents of your dream are clear as day � you see Uldum under a darkened sky, while countless armies of tol'vir march upon their enemies. But the important thing that gets your attention is that they have their original stone bodies and you notice a figure leading them. It's you, also with a body made of stone. Not too long after that dream, visitors have reached your court, offering you power and your original body in exchange for servitude for the Old Gods.;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10105;A Solution to the Curse;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10105;The agents of the Twilight's Hammer begin preparing the site for a ritual that will restore your original stone body. You wonder if this is indeed a fair proposition, but you realize that you're getting closer to your demise one way or another, and servitude in exchange for power and immortality might be a small price to pay. The cultist motions you to enter the ritual circle, and soon you're engulfed with familiar power. It seems they have harnessed the power of a mighty Titan artifact, and they begin channeling it upon you. Shortly after, you feel different. They have indeed made good on their promise. Now it's up to you to decide... to serve the Old Gods? Or to rule by yourself?;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10110;The Price of Defiance;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10110;After you have disposed of your 'benefactors', you began preparing yourself for the inevitable consolidation of power within your realm. However, the Twilight's Hammer didn't forget about your betrayal... Multiple shadowy figures converge upon your court, and now you realize that you have to fight for your life in order to keep your prized stone body. If the visions were correct, you should prevail and emerge as the sole uncontested ruler of the reborn Neferset.;;;;;;;;;;;;;x
```

Comment:
Changed the tenses in the first and second sentences.
```suggestion
EVTDESC_WCVOD_10110;After you disposed of your 'benefactors', you began preparing yourself for the inevitable consolidation of power within your realm. However, the Twilight's Hammer have not forgotten your betrayal... Multiple shadowy figures converge upon your court, and now you realize that you have to fight for your life in order to keep your prized stone body. If the visions were correct, you should prevail and emerge as the sole uncontested ruler of the reborn Neferset.;;;;;;;;;;;;;x
```

Reply:
Huh, I figured this as referring to the cult, so I thought singular was fine. But works this way too, I guess.